go.mod

```diff
@@ -21,36 +21,32 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/ipfs/go-ds-badger2 v0.1.3
 	github.com/ipfs/go-log v1.0.5
-	github.com/libp2p/go-libp2p v0.21.0
-	github.com/libp2p/go-libp2p-core v0.19.1
```

Comment:
`github.com/libp2p/go-libp2p-core` and `github.com/libp2p/go-libp2p-peerstore` are gone. They were folded into packages of `github.com/libp2p/go-libp2p`.